lms/static/sass/partials/base/_variables.scss

```diff
@@ -224,6 +224,11 @@ $success-color: rgb(0, 155, 0) !default;
 // #COLORS- EDX-SPECIFIC
 // ----------------------------
 
+// logo colors
+$micromasters-logo-blue: #005585;
+$xseries-logo-gray: #424242;
+$professional-certificate-logo-red: #9a1f60;
```

Comment:
Nitpick: I'm not sure it matters that they are logos or what the name of the color is (e.g. if next month they wanted to make the xseries color orange) so I would consider renaming them ``$micromasters-color```, etc.